### PR TITLE
DDS-1385 [Progress Bar] Retouching

### PR DESCRIPTION
**Pull request by poetrainy**
## Related Ticket(s) and Links

- JIRA Ticket: [DDS-1385](https://daikinsvoil.atlassian.net/browse/DDS-1385)
- Figma design: [Figma](https://www.figma.com/design/VyaaU8Ta9yzyf0PsURWSSf/branch/TtmQnFinEWxw7PeiggCVdJ/DDS%3A-Design-Kit?node-id=31780-356&node-type=frame&t=pPdZP4KMBEvbf0XT-0)

## Checklist

- [x] I have added a changeset file that describes the changes.
- [x] I have performed a self-review of my code.
- [x] I have added/updated appropriate regression tests.
- [x] I have confirmed that there are no accessibility warnings that can be addressed.
- [x] I have confirmed that the bug fix does not affect the keyboard accessibility of the component.
- [x] I have confirmed that the fix does not introduce any new visual regressions.

[DDS-1385]: https://daikinsvoil.atlassian.net/browse/DDS-1385?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: icons/status-error.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <path fill="currentColor" fill-rule="evenodd" d="M8 16a8 8 0 1 1 8-8 8 8 0 0 1-8 8ZM7.97 2.854c-.652 0-1.146.34-1.146.89v5.047c0 .55.494.89 1.146.89.637 0 1.147-.354 1.147-.89V3.744c0-.537-.51-.89-1.147-.89Zm-1.137 9.102c0-.628.51-1.138 1.137-1.138a1.138 1.138 0 0 1 0 2.274c-.627 0-1.137-.51-1.137-1.136Z" clip-rule="evenodd"/>
```
Comment (yodas7) [inline icons/status-error.svg:2]:
```suggestion
  <path fill="currentColor" fill-rule="evenodd" d="M8 16a8 8 0 1 1 8-8 8 8 0 0 1-8 8ZM7.97 2.854c-.652 0-1.146.34-1.146.89v5.047c0 .55.494.89 1.146.89.637 0 1.147-.354 1.147-.89V3.744c0-.537-.51-.89-1.147-.89Zm-1.137 9.102c0-.628.51-1.138 1.137-1.138a1.138 1.138 0 0 1 0 2.274c-.627 0-1.137-.51-1.137-1.136Z"/>
```

File: src/components/progress-bar/daikin-progress-bar.ts [hunk 2]
```diff
@@ -99,8 +116,13 @@ export class DaikinProgressBar extends LitElement {
...
    */
   @property({ type: String })
-  variant: "inprogress" | "completed" | "indeterminate" | "error" =
-    "inprogress";
+  variant: ProgressBarVariantProps["variant"] = "inprogress";
+
+  /**
+   * Size of the progress bar
+   */
+  @property({ type: String })
```
Comment (yodas7) [inline src/components/progress-bar/daikin-progress-bar.ts:124]:
```suggestion
  @property({ type: String, reflect: true })
```

**Review by yodas7 — COMMENTED** [review]
Add a changeset file.

**Comment by poetrainy** [comment]
After solving the accessibility issues, we will open this branch.

File: src/components/progress-bar/daikin-progress-bar.ts [hunk 3]
```diff
@@ -1,16 +1,13 @@
-import {
-  colorFeedbackNegative,
-  colorFeedbackPositive,
-} from "@daikin-oss/dds-tokens/js/daikin/Light/variables.js";
 import { cva } from "class-variance-authority";
 import { LitElement, css, html, unsafeCSS, type PropertyValues } from "lit";
 import { customElement, property } from "lit/decorators.js";
 import tailwindStyles from "../../tailwind.css?inline";
+import type { MergeVariantProps } from "../../type-utils";
+import "../icon/daikin-icon";
```
Comment (yodas7) [inline src/components/progress-bar/daikin-progress-bar.ts:6]: You don't have to import this if you don't use `<daikin-icon>` component.

**Review by yodas7 — APPROVED** [review]
LGTM

**Review by hiro-daikin — APPROVED** [review]

**Review by yodas7 — APPROVED** [review]

**Pull request merged**